.gitignore

```diff
@@ -121,6 +121,8 @@ tools/windows/install-help/cal/packages/
 
 # ebpf object files
 pkg/ebpf/bytecode/build/
+pkg/ebpf/bytecode/bindata/
+!pkg/ebpf/bytecode/bindata/bindata.go
```

Comment:
Should we remove this file also since it is generated?

Reply:
We would have to completely remove bundling then.  Without that file, even simple commands like `inv -e deps` fail.

Reply:
Nevermind. I'm not sure what the problem was.